kiwi/runtime_checker.py

```diff
@@ -134,19 +136,51 @@ def check_docker_tool_chain_installed(self):
         for those tools to be installed in the build system and fails if
         it can't find them
         """
-        message = dedent('''\n
-            Required tool {0} not found in caller environment
+        message_tool_not_found = dedent('''\n
+            Required tool {name} not found in caller environment
 
             Creation of docker images requires the tools umoci and skopeo
             to be installed on the build system. For SUSE based systems
             you can find the tools at:
 
             http://download.opensuse.org/repositories/Virtualization:/containers
         ''')
+        message_version_unsupported = dedent('''\n
+            {name} tool found in unsupported version
+
+            Expected version: v{want_version}.x.x but got: v{got_version}.x.x
+        ''')
+
+        expected_version = 0
```

Comment:
Yeah, this looks good to me. However I would wait until `umoci` hits `1.0` before doing this (since SemVer only defines such version incompatibilities after `1.0`) -- or making the check just check for `1`. *If* we make the API incompatible it will be in a `2.0` (because we're going to maintain the `1.0` release branch for a reasonable period of time).